lib/src/divider/types.ts

```diff
@@ -0,0 +1,16 @@
+type Props = {
+  /**
+   * The divider can be showed in horizontal or vertical
+   */
+  orientation?: "horizontal" | "vertical";
+  /**
+   * Modifies the thickness of the divider.
+   */
+  weight?: "regular" | "strong";
+  /**
+   * Modifies the color of the divider
+   */
+  color?: "default" | "darkGrey";
```

Comment:
I don't think that the word `default` is appropriate here. Both values are inconsistent with each other. Do we consider something like: `"mediumGrey"` and `"darkGrey"`? Also, aren't we missing a light grey, like the one in the Select component (`color-grey-200`)?

Reply:
This is currently under investigation by the designers.

Reply:
Hello... my suggestion for this if we're following this line of thought is we use lightGrey, Grey, and darkGrey

Reply:
In the end it was decided that the default option will be called mediumGrey instead of just Grey for homogeneization purposes.